phoebe/__init__.py

```diff
@@ -17,7 +17,7 @@
 
 """
 
-__version__ = '2.4.17.dev+feature-blending'
+__version__ = '2.5.0.alpha'
```

Comment:
remind me why this is necessary?  Can we not stick with the existing version tagging scheme for feature branches?

Reply:
because 2.4 and 2.5 store passband data differently, and version triggers different pathways in the code (i.e., how passband files are loaded). See https://github.com/aprsa/phoebe2/blob/ac7e009daf9fec24e0bd9f736a5863f112d55cfc/phoebe/atmospheres/passbands.py#L555 for an example.

Reply:
Can that check for the version tag until we know this will officially be in 2.5 and merge into the release branch?

Reply:
Fixed by commit https://github.com/aprsa/phoebe2/commit/a7ec893d7d9c41cc7b2f7fc10d0ee8493cc23fcc.